<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapSensor</class>
 <widget class="QMainWindow" name="MapSensor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1375</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>408</width>
    <height>601</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1375</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sensorado de tortugas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/Images/if_Turtle_204384.ico</normaloff>:/Images/if_Turtle_204384.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(57, 134, 137)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbPrimary">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>49</y>
      <width>1250</width>
      <height>581</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1250</width>
      <height>600</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(231, 231, 231);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="gbDatos">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>39</y>
       <width>451</width>
       <height>480</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1100</width>
       <height>480</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Datos de la tortuga sensorada</string>
     </property>
     <widget class="QLabel" name="lbSensor">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sensor</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cboSensor">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>291</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbIdTortuga">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Id Tortuga</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbNombreTortuga">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nombre tortuga</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbSubespecie">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Subespecie</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbInvestigador">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Investigador</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbZona">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>291</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Zona</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbMunicipio">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Municipio</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbEstado">
      <property name="geometry">
       <rect>
        <x>34</x>
        <y>370</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Estado</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneIdTortuga">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>271</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneNombreTortuga">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>170</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneSubespecie">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>210</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneInvestigador">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>250</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneZona">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>290</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneMunicipio">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>330</y>
        <width>291</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneEstado">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>370</y>
        <width>311</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="lbMensaje">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>520</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>0</y>
       <width>401</width>
       <height>561</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/Images/tortugaagua.jpeg</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pbDetener">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>390</y>
       <width>180</width>
       <height>70</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>70</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton#pbDetener {
background-color: rgb(120, 190, 151);
 }

 QPushButton#pbDetener:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
     </property>
     <property name="text">
      <string>Detener</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/Images/381031-200.ico</normaloff>:/Images/381031-200.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSensor">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>240</y>
       <width>180</width>
       <height>70</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>70</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton#pbSensor {
background-color: rgb(120, 190, 151);
 }

 QPushButton#pbSensor:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
     </property>
     <property name="text">
      <string>Activar sensor</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/Images/381031-200.ico</normaloff>:/Images/381031-200.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>100</y>
       <width>180</width>
       <height>70</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>70</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton#pushButton {
background-color: rgb(120, 190, 151);
 }

 QPushButton#pushButton:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
     </property>
     <property name="text">
      <string>Activar ventana</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/Images/switch_windows.ico</normaloff>:/Images/switch_windows.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1375</width>
      <height>700</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>gbPrimary</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
